documentation/source/contributors.rst

```diff
@@ -6,7 +6,14 @@ Contributors
    McGregor
    Grimwood
 
-cocotb was developed by `Potential Ventures <https://potential.ventures>`_ with the support of
+cocotb is developed and maintained by an active community.
+Our GitHub repository contains a list of all `contributors <https://github.com/cocotb/cocotb/graphs/contributors>`_.
+
+Since mid-2018 cocotb is an independent project under the umbrella of the
+`Free and Open Source Silicon Foundation <https://www.fossi-foundation.org>`_.
+The FOSSi Foundation provides the cocotb project with financial,
+legal and administrative support, and holds cocotb assets.
+
+cocotb was originally developed by `Potential Ventures <https://potential.ventures>`_ with the support of
```

Comment:
Sorry, another one. Please explicitly add Stu and Chris here and remove the Potential Ventures link, the company isn't active any more and the link is dead. Also remove the Solarflare link, the company has since been bought by Xilinx and a link doesn't lead anywhere useful.

> cocotb was originally developed by Stuart Hodgson and Chris Higgs of Potential Ventures with the support of Solarflare Communications Ltd.

Reply:
https://potentialventures.github.io is still around.

Reply:
Edit: no it isn't, just it's source is.

Reply:
Done.